frontend/app/src/components/dialogs/MissingRulesDialog.vue

```diff
@@ -10,7 +10,7 @@ const modelValue = defineModel<HistoryEventEditData | undefined>({ required: tru
 const emit = defineEmits<{
   'redecode': [data: LocationAndTxHash];
   'edit-event': [event: HistoryEventEditData];
-  'add': [rule: Pick<AccountingRuleEntry, 'eventType' | 'eventSubtype' | 'counterparty'>];
+  'add': [rule: Pick<AccountingRuleEntry, 'eventType' | 'eventSubtype' | 'counterparty' | 'identifier'>];
```

Comment:
minor: but since I can see we are repeating this type down in the dialog-types can we extract a type alias there and import it here so that we don't have to update it to multiple places?